codeserver/ubi9-python-3.12/nginx/root/usr/share/container-scripts/nginx/common.sh

```
@@ -13,19 +12,17 @@
-      echo "=> sourcing $filename ..."
-      # Custom file is prefered
-      if [ -f $custom_dir/$filename ]; then
-        source $custom_dir/$filename
-      elif [ -f $default_dir/$filename ]; then 
-        source $default_dir/$filename
-      fi
+ get_matched_files "$custom_dir" "$default_dir" '*.sh' | sort -u | while read -r filename; do
+  if [ "$filename" ]; then
+    echo "=> sourcing $filename ..."
+    if [ -f "$custom_dir/$filename" ]; then
+      . "$custom_dir/$filename"
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)